Gemfile.lock

```diff
@@ -787,7 +787,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.5.3p105
+   ruby 2.6.2p47
```

Comment:
@htmlboy we can not upgrade to Ruby 2.6 until we solve the problems with the corresponding Debian docker image and wkhtmltopdf-binary.

Reply:
@tramuntanal Sorry about that! Fixed.